From bfef6beb192e45e6bc4bea33c59d102ff3fc8c77 Mon Sep 17 00:00:00 2001
From: Alan Tull <r80115@freescale.com>
Date: Thu, 18 Sep 2008 14:22:20 -0500
Subject: [PATCH] ENGR00092528-1: ASoC: Don't display current WM8903 sequencer step

The value is incorrect in some circumstances.

commit b7fa3b27051a82742e25fe50183e766dae84c123 from
Wolfson linux-2.6-asoc repo
http://opensource.wolfsonmicro.com/

Signed-off-by: Alan Tull <r80115@freescale.com>
---
 sound/soc/codecs/wm8903.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/sound/soc/codecs/wm8903.c b/sound/soc/codecs/wm8903.c
index b16a81b..3e4451f 100644
--- a/sound/soc/codecs/wm8903.c
+++ b/sound/soc/codecs/wm8903.c
@@ -365,12 +365,6 @@ static int wm8903_run_sequence(struct snd_soc_codec *codec, unsigned int start)
 		msleep(10);
 
 		reg[4] = wm8903_read(codec, WM8903_WRITE_SEQUENCER_4);
-
-		if (reg[4] & WM8903_WSEQ_BUSY)
-			dev_dbg(&wm8903_i2c_client->dev,
-				"Sequence running at step %d\n",
-				(reg[4] >> WM8903_WSEQ_CURRENT_INDEX_SHIFT) & 0x1f);
-
 	} while (reg[4] & WM8903_WSEQ_BUSY);
 
 	dev_dbg(&wm8903_i2c_client->dev, "Sequence complete\n");
-- 
1.5.4.4

